src/classes/CON_PrimaryAffls_TDTM.cls

```diff
@@ -32,14 +32,14 @@
 * @date 2015
 * @group Contacts
 * @group-content ../../ApexDocContent/Contacts.htm
-* @description Creates Affiliations when the user manually populates
-* any of the primary affiliation lookup fields.
+* @description
+1) Creates Primary Affiliations when the user populates any of the Primary Affiliation lookup fields on the Contact object.
```

Comment:
Also double checked the spec doc, it looks like this has not been reviewed by @juliesnow. Imo it should since it is a TDTM class. Forget it if you have thou!

Reply:
Updated styling, I have let @juliesnow know about this change on the quip doc earlier.